packages/auth/src/app.ts

```diff
@@ -187,6 +204,27 @@ export class App {
       }
     }
 
+    // Interaction
+    this.publicRouter.get(
+      '/interact/:interactId',
+      grantRoutes.interaction.start
+    )
+
+    this.publicRouter.post(
+      '/interact/:interactId/login',
```

Comment:
Why do you want to add the `/login`?

Reply:
I think we can just leave it. We have the `GET` to this url, which is for `start`, and the `POST`, which is for `finish`. At least that is how I speced it out [here](https://github.com/interledger/open-payments/pull/141).